src/styles/framework.scss

```diff
@@ -200,32 +200,33 @@ span > a,
     &:not(.button):not(.no-link-decoration) {
         position: relative;
         text-decoration: none;
-        display: inline-block;
-        padding: 0 0px;
         @media (prefers-reduced-motion: no-preference) {
-            transition: color ease 0.3s;
+            transition: all ease 0.3s;
         }
 
-        &::after {
-            content: '';
-            position: absolute;
-            z-index: -1;
-            width: 100%;
-            height: 2px;
-            left: 0;
-            bottom: 0;
-            background-color: $accent-color;
-            @media (prefers-reduced-motion: no-preference) {
-                transition: all ease 0.3s;
+        // Very loosely inspired by: https://codepen.io/aaroniker/pen/pojaBvb
+        background-image: linear-gradient(0deg, $accent-color 0%, $accent-color 100%);
+        background-position: bottom left;
+        background-repeat: no-repeat;
+        background-size: 100% 2px;
+        $external-link-selector: '&:not([href^="https://www.datenanfragen.de"]):not([href^="https://www.datarequests.org"]):not([href^="https://www.demandetesdonnees.fr"]):not([href^="https://www.pedidodedados.org"]):not([href^="https://static.dacdn.de"]):not([href^="http://localhost:"]):not([href^="http://datenanfragen.test:"]):not([href^="https://verein.datenanfragen.de"]):not([href^="#"]):not([href*="--datenanfragen.netlify.app/"])';
+
+        #{$external-link-selector} {
+            background-size: calc(100% - 16.8px) 2px;
+            &:after {
+                color: $accent-color;
+                font-icon: 80% url('external.svg');
+                padding-left: 4px;
+                vertical-align: -0.5px;
             }
         }
 
         &:hover,
         &:focus {
             color: color('white');
-
-            &::after {
-                height: 100%;
+            background-size: 100% 100%;
+            #{$external-link-selector} {
+                background-size: calc(100% - 16.8px) 100%;
```

Comment:
Why exclude the symbol from the background?

Reply:
This way it is not affected by the hover animation. Otherwise the animation would cover the symbol as well, which would defeat its purpose, if we do not change its color, and does not look good imo.

Reply:
I agree with @mal-tee that including the symbol in the underline looks worse. See also the reference provided in #287.